.changeset/dry-lemons-doubt.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/perseus": minor
+---
+
+Display tooltip for movable point when option is indicated
```

Comment:
Our changelogs aren't critically important, but I think it'd be good to clarify here that we're talking about the `interactive-graph` widget.